src/nidm/experiment/Utils.py

```diff
@@ -1870,13 +1870,13 @@ def map_variables_to_terms(
                                 )
 
                         print(
-                            f"isAbout: @id = {column_to_terms[current_tuple]['isAbout']['@id']}, label = {column_to_terms[current_tuple]['isAbout']['label']}"
+                            f"isAbout: @id = {column_to_terms[current_tuple]['isAbout'][0]['@id']}, label = {column_to_terms[current_tuple]['isAbout'][0]['label']}"
```

Comment:
we figured that it was a wrong thing to do -- there could be lists and scalars

Reply:
We were testing on an older version of the json sidecar files. All isAbout entries in either BIDS-compliant JSON or NIDM-compliant JSON-LD sidecar files use a list of dictionaries now.